_posts/2018-05-15-Rust-turns-three.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "Rust turns three"
+author: Aaron Turon
+description: "Three years ago today, the Rust community released Rust 1.0 to the world, with our initial vision of fearless systems programming."
+---
+
+# Rust turns three
+
+Three years ago today, the Rust community released Rust 1.0 to the world, with
+our initial vision of fearless systems programming. As per tradition, we’ll
+celebrate Rust’s birthday by taking stock of the people and the product, and
+especially of what’s happened in the last year.
+
+# The People
+
+Rust is a people-centric, consensus-driven project. Some of the most exciting
+developments over the last year have to do with how the project itself has
+grown, and how its processes have scaled.
+
+The [official teams](https://www.rust-lang.org/en-US/team.html) that oversee the
+project **doubled** in size in the last year; there are now over a hundred
+individuals associated with one or more of the teams. To accommodate this scale,
+the team structure itself has evolved. We have top-level teams covering the
+language, library ecosystem, developer tooling, documentation, community, and
+project operations. Nested within these are dozens of subteams and working
+groups focused on specific topics.
+
+Rust is now used in a huge variety of companies, including both newcomers and
+big names like Google, Facebook, Twitter, Dropbox, Microsoft, Redhat, npm and,
```

Comment:
Are we going to link to anything from here?  Btw, we should probably drop Microsoft since the only public thing I can think of (though please correct me if I'm wrong) is the search functionality in vscode.

Reply:
Why is the VSCode thing not enough?

Reply:
They aren't using Rust.  They're just using a tool written in Rust.  It's not enough, imho, from that to claim "Microsoft is using Rust" in a callout like this.

Reply:
They're putting Rust code inside of software that they write and distribute.

But beyond that, there's also that repo I linked, which is also Rust code written directly by microsoft.

Reply:
An interesting thing, on this note: https://www.reddit.com/r/rust/comments/8jnb04/rust_turns_three/dz168xi/?context=4